src/main/java/model/Money.java

```diff
@@ -0,0 +1,25 @@
+package model;
+
+import java.math.BigDecimal;
+
+public class Money {
+    private static final String MONEY_MUST_BE_POSITIVE = "구입금액은 0 이상의 숫자여야 합니다.";
+    private static final int ZERO = 0;
+
+    private final BigDecimal amount;
+
+    public Money(BigDecimal amount) {
+        validateMoney(amount);
+        this.amount = amount;
+    }
+
+    public BigDecimal getAmount() {
+        return amount;
+    }
+
+    private void validateMoney(BigDecimal amount) {
+        if (amount.compareTo(BigDecimal.ZERO) < ZERO) {
```

Comment:
``ZERO`` 라는 의미가 약간 모호한 것 같아요. 
``compareTo`` 은  A > B, A == B, A < B 인 경우 각각 1, 0, -1 의 값이 나오는데, 

```suggestion
        if (amount.compareTo(BigDecimal.ZERO)  != -1) {
```
개인적인 의견이지만 위와 같이 변경해서, -1을 상수로 추출하여 의미있는 네이밍을 부여한다면? 조금 더 의미가 와닿지 않을까요 🤔?

Reply:
고민해서 반영하겠습니다!